debian/libmtbl1.symbols

```diff
@@ -3,6 +3,7 @@ libmtbl.so.1 libmtbl1 #MINVER#
  LIBMTBL_1.1.0@LIBMTBL_1.1.0 1.1.0
  LIBMTBL_1.2.0@LIBMTBL_1.2.0 1.3.0
  LIBMTBL_1.4.0@LIBMTBL_1.4.0 1.6.0
+ LIBMTBL_1.5.0@LIBMTBL_1.5.0 1.7.0
```

Comment:
Allan suggests something might be off in the numbering here vs libmtbl.sym.
A reference is https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html
but Allan or Jeremy can fix it when they process the merge from next to master.

Reply:
Whoops, I believed these are fixed now.